Makefile

```diff
@@ -380,30 +365,35 @@ create-ocp-manifests:
 	export HW_REQUIREMENTS="$(subst ",\", $(shell cat $(ROOT_DIR)/data/default_hw_requirements.json | tr -d "\n\t "))" && \
 	$(MAKE) deploy-postgres deploy-ocm-secret deploy-s3-secret deploy-service deploy-ui
 
-ci-deploy-for-subsystem: _verify_cluster generate-keys
-	export TEST_FLAGS=--subsystem-test && export AUTH_TYPE="rhsso" && export DUMMY_IGNITION=${DUMMY_IGNITION} && \
-	export IPV6_SUPPORT="True" && export ENABLE_ORG_TENANCY="True" && export ENABLE_ORG_BASED_FEATURE_GATES="True" && \
-	export RELEASE_SOURCES='$(or ${RELEASE_SOURCES},${DEFAULT_RELEASE_SOURCES})' && \
-	$(MAKE) deploy-wiremock deploy-all
+ci-deploy-for-subsystem: deploy-test
 
-patch-service: _verify_cluster update-local-image
+patch-service: _verify_cluster
+	$(MAKE) load-image SERVICE=localhost/assisted-service:latest
```

Comment:
Why $(MAKE) ? Can't this be added as dependency as `_very_cluster` target is?

Reply:
can't we name it as the user specifies and use the global SERVICE variable instead? Is the problem only replacing the manifest's value?

Reply:
I prefer not changing it globally as it is used in many targets and it is hard to anticipate the scope of its impact

Reply:
But this way the user is forced to build an image  all the time, right? i.e. they wouldn't be able to quickly use something they pushed to quay.io without editing the config, right?